LDMP/select_dataset.py

```diff
@@ -62,16 +62,11 @@ def lc_band(self):
     def soil_band(self):
         return self.combo_soil.get_current_band()
 
-    def sdg_band(self):
```

Comment:
Why are you removing this function? It is still used in `calculate_ldn.py` and `jobs/mvc.py`.

Reply:
@azvoleff Removed it because its a duplicate function (see line 53). See https://github.com/ConservationInternational/trends.earth/blob/f88fae8601ea281db3f5cd88da6d86ceaa56b9f1/LDMP/select_dataset.py#L53. So the function is still there, and it will cause no issues.

Reply:
Whoops - sorry, I read through too fast.